src/lib/Gen/xml/XMLWriter.cpp

```diff
@@ -381,6 +385,89 @@ writeGuide(
     closeTemplate(I.Template);
 }
 
+void
+XMLWriter::
+writeAlias(
+    AliasInfo const& I)
+{
+    tags_.open(aliasTagName, {
+        { "name", I.Name },
+        { I.Access },
+        { I.id }
+        });
+
+    writeSourceInfo(I);
+
+    writeJavadoc(I.javadoc);
+
+    if (I.AliasedSymbol)
+    {
+        Attributes nameAttrs = {};
+        if (I.AliasedSymbol->id != SymbolID::invalid)
+        {
+            nameAttrs.push({"id", toString(I.AliasedSymbol->id)});
```

Comment:
Don't call `toString` on the `SymbolID`, just write `nameAttrs.push({"id", I.AliasedSymbol->id});`. Also, remove the `if (I.AliasedSymbol->id != SymbolID::invalid)` condition this statement is in.